src/components/camera/show-page-card.tsx

```diff
@@ -74,21 +77,30 @@ export const CameraShowPageCard = ({
   device: CameraDevice;
   facilityId: string;
 }) => {
+  const [user] = useAtom(userAtom);
+  const careUsername = user.username ?? "";
```

Comment:
can't we keep this usage of atom inside the `CameraFeedWatermark` component itslef? 🤔 

Let's avoid polluting this component unless absolutely needed